File: java_dic/app/src/main/java/com/Dictionary/Word.java
package com.Dictionary;

public class Word {


    public String key;
    public String value;
    public int dicType;

    public Word(){


    }

    public Word(String key,String value){
        this.key = key;
        this.value = value;


    }
    public Word(String key,String value,int type){
        this.key = key;
        this.value = value;
        this.dicType = type;


    }



}

File: flutter_dic/lib/features/search/domain/entities/word.dart
import 'package:equatable/equatable.dart';

class Word extends Equatable {
  final String key;
  final String value;
  final String dicType;

  const Word({
    this.key = '',
    this.value = '',
    this.dicType = '',
  });

  @override
  List<Object?> get props => <Object?>[key, value, dicType];
}